packages/web-components/fast-components/src/anchor/index.ts

```diff
@@ -25,7 +25,7 @@ export class Anchor extends FoundationAnchor {
      * HTML Attribute: appearance
      */
     @attr
-    public appearance: AnchorAppearance;
+    public appearance: AnchorAppearance = "neutral";
```

Comment:
For some reason I thought we couldn't do this and we had to check in the connectedCallback. Maybe it was an old bug that was fixed...I though there was some sort of timing issue with upgrades or binding. Something like that. @nicholasrice Does any of this ring a bell for you?

Reply:
hmmm maybe? I didn't see the classList mutation, that may be why it was done this way. It is probably best to keep the assignment in the property initialization and just move the class to the template.